Models/ReverseProxySettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Fenrus.Models;
+
```

Comment:
add xml comments to the class/properties

Reply:
I added comments